packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -200,50 +201,51 @@ export class AtomicSearchBox {
           onMouseDown={(e) => e.preventDefault()}
           part="suggestion"
           id={id}
-          class="suggestion h-9 px-2 cursor-pointer text-left text-sm bg-transparent border-none shadow-none hover:bg-primary hover:text-on-primary flex flex-row items-center"
+          class="suggestion h-9 px-2 cursor-pointer text-left text-sm bg-transparent border-none shadow-none flex flex-row items-center"
           innerHTML={suggestion.highlightedValue}
           value={index}
         />
       );
     });
   }
 
-  private renderInputWrapper() {
-    let roundedClasses = this.leadingSubmitButton
-      ? 'border-l-0 rounded-r-lg'
-      : 'border-r-0 rounded-l-lg';
-
-    if (this.searchBoxState.suggestions.length) {
-      roundedClasses += ' rounded-bl-none rounded-br-none';
-    }
-
-    return (
-      <div
-        part="input-wrapper"
-        class={`input-wrapper flex flex-grow items-center border border-divider ${roundedClasses}`}
-        ref={(el) => (this.containerRef = el as HTMLElement)}
-      >
-        {this.input}
-        {this.clearButton}
-      </div>
-    );
-  }
-
   public render() {
     return (
-      <div class="relative">
-        <div class="search-box box-border w-full flex">
+      <div class="search-box relative w-full lg:w-80">
```

Comment:
There's a lot of things happening in that function, I feel it would be easier to read by just splitting it up a bit in logical section.

```
renderWrapper()
renderCombobox()
renderSuggestions()
```

Maybe even just extracting the `class` generation to a separate function would be good enough.

eg:
```jsx
<div class={this.searchBoxWrapperClasses}>

     {this.leadingSubmitButton && this.submitButton}

     <div class={this.comboboxClasses}>

            <div class={this.inputWrapperClasses} ref={(el) => (this.containerRef = el as HTMLElement)}>
              {this.input}
              {this.clearButton}
            </div>
</div>


etc..
```